<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewOrderWindow</class>
 <widget class="QWidget" name="NewOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1441</width>
     <height>991</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background {  
    border-image:url(:/icons/background.png) 0 0 0 0 stretch stretch;
	background-size: contain;  
}</string>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>返  回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkoutButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>添  加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>饮 品 名 称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>饮 品 咖 啡 因 含 量</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>970</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>饮 品 名 称</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
